src/test/kotlin/net/maizegenetics/phgv2/utils/MAFUtils.kt

```diff
@@ -0,0 +1,93 @@
+package net.maizegenetics.phgv2.utils
+
+import biokotlin.util.bufferedReader
+import java.io.BufferedWriter
+import java.io.FileWriter
+
+data class MAFRecord(val chr: String, val start: Int, val length: Int, val strand: String, val sequence: String)
+
+fun testMergingMAF(inputFile: String, outputFile: String) {
+
+    val reader = bufferedReader(inputFile)
+    // val reader = bufferedReader("/Users/zrm22/Downloads/LineB.maf.orig")
```

Comment:
Can you remove these comments.  Not sure how they are still in the tests...

Reply:
Ok, I removed them